.gitlab/lassen-jobs.yml

```
@@ -5,69 +5,100 @@
 # SPDX-License-Identifier: (MIT)
 ##############################################################################
 
-####
-# All lassen build jobs
+# CPU ONLY
+ibm_clang_9:
+  variables:
+    SPEC: "+fortran %clang@9.0.0ibm"
+  extends: .build_and_test_on_lassen
 
-clang_default (build and test on lassen):
+ibm_clang_9_gcc_8:
   variables:
-    SPEC: "%clang@default"
+    SPEC: "+fortran %clang@9.0.0ibm cxxflags=--gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1 cflags=--gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1"
   extends: .build_and_test_on_lassen
 
-gcc_default (build and test on lassen):
+gcc_8_3_1:
   variables:
-    SPEC: "%gcc@default"
+    SPEC: "+fortran%gcc@8.3.1"
```

Do spaces matter?  Is there a difference between what is there and
```suggestion
    SPEC: "+fortran %gcc@8.3.1"
```